docs/weaviate/manage-collections/collection-aliases.mdx

```diff
@@ -0,0 +1,330 @@
+      endMarker="# END ListCollectionAliases"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## Get a specific alias
+
+Retrieve information about a specific alias.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START GetAlias"
+      endMarker="# END GetAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+## Update an alias
+
+Change the target collection that an alias points to. This operation is atomic and provides instant switching between collections.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START UpdateAlias"
+      endMarker="# END UpdateAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+:::tip Use case: Zero-downtime migration
+Updating an alias is particularly useful for migrations:
+
+1. Create a new collection with updated schema
+2. Import data into the new collection
+3. Update the alias to point to the new collection
+4. All queries using the alias now seamlessly use the new collection
+   :::
+
+## Delete an alias
+
+Remove an alias. This only deletes the alias pointer, not the underlying collection.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START DeleteAlias"
+      endMarker="# END DeleteAlias"
+      language="py"
+    />
+  </TabItem>
+
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+:::note
+
+- Deleting a collection does not automatically delete aliases pointing to it
+  :::
+
+## Using aliases in operations
+
+Once created, aliases can be used anywhere a collection name is expected:
```

Comment:
Is it that an alias can be used where you might **work with** a collection? I think saying aliases `can be used anywhere a collection name is expected` is ambiguous and not quite true.

Do things like `client.collections.update` work? (I haven't tried it)

Reply:
Seems only collection deletion cannot be performed with the alias. Changed the wording a bit